torchhd/datasets/collection_datasets.py

```diff
@@ -0,0 +1,403 @@
+import os
+import os.path
+from typing import Callable, Optional, Tuple, List
+import torch
+from torch.utils import data
+import pandas as pd
+import tarfile
+import numpy as np
+
+from .utils import download_file_from_google_drive
```

Comment:
Could you rename this file to just `torchhd/datasets/dataset.py` then this file can contain all the abstract base datasets classes.

Reply:
Done